app/models/office_post_code.rb

```diff
@@ -0,0 +1,12 @@
+# An office can have many office post codes, which are the post codes (or partial)
+# that the office serves.
+# This allows a lookup (using with_partial_match) when a claim is submitted, so the system
+# can decide which office it is to be sent to.
+class OfficePostCode < ApplicationRecord
+  belongs_to :office
+
+  scope :active, -> { where('postcode is not null') }
```

Comment:
you can do where.not(postcode: nil) so you don't have to use SQL in your query.

Reply:
nice one - will change to that